eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -17,7 +17,7 @@ steps:
         --kusto-url-var KUSTO_URL `
         --kusto-database-var KUSTO_DB `
         --kusto-table-var KUSTO_TABLE `
-        --identity "$(Build.QueuedBy)" `
+        --identity "$(Build.RequestedForEmail)" `
```

Comment:
We sure this works with github? Looks like it might be specific to Azure Repos, see https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#how-are-the-identity-variables-set.

Reply:
May be I should use both username and email.